src/scss/constants.scss

```diff
@@ -69,6 +69,7 @@ $send-msg-online-color: #78b553;
 
 $map-sidebar-width: 20%;
 $navbar-height: 66px;
+$navbar-search-height: 54px;
```

Comment:
I'd suggest to open a refactoring PR to split the `scss/constants.scss` into several ones. The file has become `_global2.scss` by this point.

If `$navbar-search-height` is kept there, it's right next to related one. If, however moved to `NavBar.scss` (another bloated file), I can suggest making it the difference between the above `$navbar-height` and the top and bottom margin. Example
```scss
$navbar-search-height: $navbar-height - 2 * $spacing--sm;
```

Reply:
According to a followup comment https://github.com/sparkletown/sparkle/pull/2010#discussion_r695161002 this or similar variable can be used to make all the elements in the navbar same size.

@alexey-petrov using multiple classes on the same element is what CSS was made for, so splitting up the height into a utility modifier, let's say `NavBar--normal-size` with the above variable and putting that one on all the elements that require it can be the way to go.

Reply:
removed `$navbar-search-height`completely 0700a1d